### PR TITLE
Rohan's Dashboard complete with check determined on login

### PR DESCRIPTION
**Pull request by Cameron-Yoffe**
Updated Dashboard with the ability to check who is logged in dependent upon role. Members see member dashboard, admins (presidents and officers) see admin dashboard.

### CODE REVIEW

File: app/views/dashboards/show.html.erb [hunk 1]
```diff
@@ -1,14 +1,94 @@
...
+      toggleButtons.forEach(function(button) {
+        button.addEventListener('click', function() {
+          console.log(adminInfo.style.display); 
+          adminLinks.style.display = adminLinks.style.display === 'none' ? 'block' : 'none';
+          if(adminInfo.style.display == 'none' || adminInfo.style.display == ''){
+            adminInfo.style.display = 'block'; 
+          }
+        else{
+          adminInfo.style.display = "none"; 
+
```
Comment (Carriez0117) [inline app/views/dashboards/show.html.erb:87]: Remove white space

**Comment by Rohan889** [comment]
This is wonderful code and hopefully the coder continues to do his good work.

File: app/views/dashboards/show.html.erb [hunk 2]
```diff
@@ -1,14 +1,94 @@
...
+          <%= link_to 'View Officer Members', officer_index_members_path %><br>
+        </div>
+      </div>  
+    <% end %>
+  <% end %>
+</div>
+
+
+  <script>
+
```
Comment (Carriez0117) [inline app/views/dashboards/show.html.erb:71]: Remove white space

**Review by Carriez0117 — APPROVED** [review]
Remove white spaces

**Pull request merged**